cmd/query/app/static_handler.go

```diff
@@ -87,9 +89,12 @@ type loadedConfig struct {
 func NewStaticAssetsHandler(staticAssetsRoot string, options StaticAssetsHandlerOptions) (*StaticAssetsHandler, error) {
 	assetsFS := ui.StaticFiles
 	if staticAssetsRoot != "" {
-		assetsFS = http.Dir(staticAssetsRoot)
+		uiBasePath := options.UIBasePath
```

Comment:
ok, it's pointless for me to review beyond this point until you write clear definition what each of the two params means and and how it should be used. The code below is wrong.